config/locales/client.en.yml

```diff
@@ -39,9 +39,38 @@ en:
               description: "Either gpt-4 or gpt-3-5-turbo or claude-2"
 
     discourse_ai:
+      title: "AI"
       modals:
         select_option: "Select an option..."
 
+      ai-persona:
```

Comment:
Didn't catch this before, but this should be `ai_persona`

Reply:
oh my :) I guess its a big editing spree ... will take care